File: go/raindrops/raindrops.go
package raindrops

import (
	"strconv"
	"strings"
)

// Convert returns string based on int
func Convert(n int) string {
	words := strings.Builder{}
	if n%3 == 0 {
		words.WriteString("Pling")
	}
	if n%5 == 0 {
		words.WriteString("Plang")
	}
	if n%7 == 0 {
		words.WriteString("Plong")
	}
	if words.String() != "" {
		return words.String()
	}
	return strconv.Itoa(n)
}

File: javascript/raindrops/raindrops.js
export const convert = num => {
  let word = "";
  if (num % 3 === 0) {
    word += "Pling";
  }
  if (num % 5 === 0) {
    word += "Plang";
  }
  if (num % 7 === 0) {
    word += "Plong";
  }
  if (word === "") {
    return String(num);
  }
  return word;
};
